assets/src/edit-story/app/prepublish/guidance/media.js

```diff
@@ -148,13 +147,8 @@ export function videoElementOptimized(element = {}) {
   const idResource = element.resource?.id;
   const idOrigin = idResource?.toString().split(':')?.[0];
   const isCoverrMedia = idOrigin === 'media/coverr';
-  const videoArea =
-    (element.resource?.sizes?.full?.height ?? element.resource?.height ?? 0) *
-    (element.resource?.sizes?.full?.width ?? element.resource?.width ?? 0);
```

Comment:
@littlemilkstudio This line is wrong. `sizes.full` is NEVER set for a video. Sizes is only a thing for images. Height and width are always set. See this two examples. 

### Example video resource data
<img width="582" alt="Screenshot 2021-05-06 at 17 02 07" src="https://user-images.githubusercontent.com/237508/117329859-fc828a00-ae8c-11eb-849c-9ce0b0e679c1.png">

### Example image resource data
<img width="727" alt="Screenshot 2021-05-06 at 17 04 35" src="https://user-images.githubusercontent.com/237508/117330015-2b006500-ae8d-11eb-9593-582533982d57.png">

Reply:
Awesome. I'm gonna update the check to not look at `sizes.full` at all. Shouldn't effect any behavior tho because it's defaulting to `resource.height` & `resource.width` if `sizes.full` is not present, which as you mentioned isn't.

But cool. thanks for the note. shouldn't effect bug bash at all today, but will cleanup.

Reply:
@spacedmonkey cleaned up here:
https://github.com/google/web-stories-wp/pull/7405

Reply:
thanks again for combing through this and pointing it out 🙏